### PR TITLE
Campaign page redesign

### PR DESCRIPTION
**Pull request by stellenberger**
So here is the campaign v12 design implemented. I have also done the footer, so that should also be applicable on other pages, too. 

One issue is at the bottom of the header image there is a thin black border from the resulting black background of the div.

### CODE REVIEW

File: src/components/app_download/AppDownload.module.scss [hunk 1]
```diff
@@ -54,3 +54,22 @@
   right: -30px;
   z-index: 0;
 }
+.downloadBtns {
+  display: flex;
+  flex-flow: wrap;
+  justify-content: center;
+  width: 50%;
+  z-index: 2;
+  & button {
```
Comment (JElgar) [inline src/components/app_download/AppDownload.module.scss:63]: Thi is a similar comment to the one on pr #58 buuut
& a is the same as just having a (when nested and with a space) and I'd probably say not having them is a bit neater.
```suggestion
  button {
```

File: src/components/app_download/AppDownload.module.scss [hunk 2]
```diff
@@ -54,3 +54,22 @@
...
+    width: 40%;
+    max-width: 170px;
+    margin-right: 16px;
+  }
+  & img {
```
Comment (JElgar) [inline src/components/app_download/AppDownload.module.scss:68]: See above
```suggestion
  img {
```

File: src/components/footer/Footer.jsx [hunk 3]
```diff
@@ -1,35 +1,79 @@
 import React from 'react';
 import classes from './Footer.module.scss';
-import { privacyNotice } from '../../assets';
+// import { nowUOrange } from '../../assets';
```
Comment (JElgar) [inline src/components/footer/Footer.jsx:3]: And similarly to #58 again, I think to keep things clean let's remove any unneeded code where possible

File: src/components/footer/Footer.jsx [hunk 4]
```diff
@@ -1,35 +1,79 @@
...
+          </div>
+          <div className={classes.linksColumnTwo}>
+            <div className={classes.links}>
+              <a href="faqs" className={classes.links}>FAQ</a>
+            </div>
+            <div className={classes.links}>
+              <a href="/" className={classes.links}>Blog</a>
+            </div>
+            <div className={classes.links}>
+              <a href="/" className={classes.links}>Newsletter</a>
```
Comment (JElgar) [inline src/components/footer/Footer.jsx:40]: This should trigger the newsletter pop up - not sure how we want to do this tho? What should we show if we know the user has already signed up? Might not be a this pr issue tho, so feel free to create a new issue for this

File: src/components/footer/Footer.module.scss [hunk 5]
```diff
@@ -1,30 +1,75 @@
...
+    text-align: center;
+  }
+  .footerRight {
+    width: 100%;
+  }
+  .logo {
+    margin: auto;
+    padding: 50px 0 0 0  
   }
 }
```
Comment (JElgar) [inline src/components/footer/Footer.module.scss:75]: This is again similar to the comment on #58 but I think nesting really helps keep the code clean (feel free to say Im wrong). But in this case I would have media queries done in the same way as @tunangst has done in his update so: 
```
.profile-pic {
  float: left;
  width: 250px;
  @media screen and (max-width: $break-small) {
    width: 100px;
    float: none;
  }
```
which compiles to:
```
.profile-pic {
  float: left;
  width: 250px;
}
@media screen and (max-width: 320px) {
  .profile-pic {
    width: 100px;
    float: none;
  }
}
```
http://thesassway.com/intermediate/responsive-web-design-in-sass-using-media-queries-in-sass-32

Similarly, I think things like .footerRight and .footLeft if those are always children of .footer then tose can be nested within the .footer block

File: src/domain/campaigns/campaign_details/CampaignDetails.component.jsx [hunk 6]
```diff
@@ -2,26 +2,50 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import ReactPlayer from 'react-player';
-import { AppDownload, Text } from '../../../components';
+// import { AppDownload, Text } from '../../../components';
```
Comment (JElgar) [inline src/domain/campaigns/campaign_details/CampaignDetails.component.jsx:5]: Same as above
```suggestion
```

File: src/domain/campaigns/campaign_details/CampaignDetails.component.jsx [hunk 7]
```diff
@@ -31,17 +55,14 @@ const Campaign = ({
...
           </div>
         )}
       </div>
       <div className={classes.contentGroup}>
-        <Text className={classes.subtitle} type="h2">
-          What is it about?
-        </Text>
         {/* <Text type="p" className={classes.caption}>
           This is a caption that concludes the key message of the video
         </Text> */}
```
Comment (JElgar) [inline src/domain/campaigns/campaign_details/CampaignDetails.component.jsx:68]: Same as above
```suggestion
```

File: src/domain/campaigns/campaign_details/CampaignDetails.component.jsx [hunk 8]
```diff
@@ -53,26 +74,33 @@ const Campaign = ({
...
         })}
+        <div className={classes.actionFooter}>
+          <Text type="p">
+            Find out what other actions you can take to support this campaign in our app!
+          </Text>
+          <AppDownloadButtons />
+        </div>
       </div>
-      <AppDownload />
+      {/* <AppDownload /> */}
```
Comment (JElgar) [inline src/domain/campaigns/campaign_details/CampaignDetails.component.jsx:102]:
```suggestion
```

File: src/domain/campaigns/campaign_details/CampaignDetails.module.scss [hunk 9]
```diff
@@ -20,19 +20,52 @@
...
 .header {
+  position: relative;
+  display: inline-block;
   height: auto;
   max-width: $max-width;
   margin: auto;
-  width: calc(100% - 36px);
-  padding: 18px;
+  width: calc(100%);
   & img {
```
Comment (JElgar) [inline src/domain/campaigns/campaign_details/CampaignDetails.module.scss:31]:
```suggestion
  img {
```

File: src/domain/campaigns/campaign_details/CampaignDetails.module.scss [hunk 10]
```diff
@@ -20,19 +20,52 @@
...
     height: 100%;
     object-fit: cover;
-    max-height: 550px;
+    max-height: 300px;
+  }
+}
+
+.headerImage {
+  background-color: black;
+  & img {
```
Comment (JElgar) [inline src/domain/campaigns/campaign_details/CampaignDetails.module.scss:41]:
```suggestion
  img {
```

File: src/domain/campaigns/campaign_details/CampaignDetails.module.scss [hunk 11]
```diff
@@ -20,19 +20,52 @@
...
+    }
+  }
+  @media screen and (max-width: 600px) {
+    top: calc(25%);
+    h1 {
+      font-size: 35px;
+    }
+  }
+}
+
```
Comment (JElgar) [inline src/domain/campaigns/campaign_details/CampaignDetails.module.scss:68]: Personally I much prefer this!

File: src/domain/campaigns/campaign_details/CampaignDetails.module.scss [hunk 12]
```diff
@@ -114,11 +153,32 @@
...
+  margin-top: 7px;
+  margin-right: 20px;
+  height: 70px;
+}
 
 .actionItem {
   display: flex;
   background: white;
-  box-shadow: $campaignBoxShadow;
+  // box-shadow: $campaignBoxShadow;
```
Comment (JElgar) [inline src/domain/campaigns/campaign_details/CampaignDetails.module.scss:181]:
```suggestion
```

File: src/domain/campaigns/campaign_details/CampaignDetails.module.scss [hunk 13]
```diff
@@ -114,11 +153,32 @@
...
+  p {
+    width: 50%;
+    margin-left: 25%;
+    font-size: 18px;
+  }
+  
+  
+}
+
+.actionIconContainer {
```
Comment (JElgar) [inline src/domain/campaigns/campaign_details/CampaignDetails.module.scss:168]: Again I think this sass section is pretty flat and would probably benefit from some more nesting

File: src/domain/campaigns/campaign_details/CampaignDetails.module.scss [hunk 14]
```diff
@@ -105,6 +139,11 @@
   margin-bottom: 48px;
 }
 .actionContainer {
+  background-color: #F7F8FC;
+  width: 100%;
+}
+
+.actionContent {
   @extend .contentGroup;
```
Comment (JElgar) [inline src/domain/campaigns/campaign_details/CampaignDetails.module.scss:147]: I didn't know this existed but that's super cool

**Comment by JElgar** [comment]
Oh I just read this "One issue is at the bottom of the header image there is a thin black border from the resulting black background of the div.". Shows I haven't done a very good job of reviewing the pr :laughing:. 
If you add `max-height: 300px;` to the black thing that seems to do the trick

**Comment by JElgar** [comment]
One other thing is, is there any way we can keep the videos aspect ration 16:9? Opposed to having this stretching on mobile (or large displays) 
Screenshot from 2020-07-28 23-38-25

File: src/components/footer/Footer.jsx (hunk 4 above)
Comment (stellenberger) [reply src/components/footer/Footer.jsx:40]: Perhaps a 'Youve already signed up!" message - Maybe we can speak to the design team about this? And perhaps also an option to resend/resignup to the newsletter if they think the first signup didnt work, like ' didnt get your last newsletter? resend newsletter'

File: src/components/footer/Footer.module.scss (hunk 5 above)
Comment (stellenberger) [reply src/components/footer/Footer.module.scss:75]: I am happy to nest the @media inside of their respective elements - though I find when writing the CSS for dynamic page sizes, its easier to have all the css code together for each screen size (if there are multiple) as it makes it easier to adjust?

Comment (stellenberger) [reply src/components/footer/Footer.module.scss:75]: So, I am not sure if this has been discovered but if you nest the css elements, the responsive media size cant read it (even if theyre nested in there, as well), so we will probably have to keep it separated. I hope that is OK!

Comment (stellenberger) [reply src/components/footer/Footer.module.scss:75]: If you find a way around this PLEASE let me know as im interested in how this works!!!

File: src/domain/campaigns/campaign_details/CampaignDetails.component.jsx [hunk 15]
```diff
@@ -62,7 +62,12 @@ const Campaign = ({
         )}
       </div>
       <div className={classes.contentGroup}>
-        <ReactPlayer className={classes.video} url={video_link} width="100%" height="45vh" />
+        {/* <div className={classes.videoWrapper}>
+          <iframe width="560" height="349" src={video_link} frameBorder="0" allowFullScreen></iframe>
+        </div> */}
```
Comment (JElgar) [inline src/domain/campaigns/campaign_details/CampaignDetails.component.jsx:67]: If possible id still get rid of commented out code

```suggestion
```

**Pull request merged**